solc/CommandLineInterface.cpp

```diff
@@ -743,232 +754,186 @@ at standard output or in files in the output directory, if specified.
 Imports are automatically read from the filesystem, but it is also possible to
 remap paths using the context:prefix=path syntax.
 Example:
-solc --)" + g_argBinary + R"( -o /tmp/solcoutput dapp-bin=/usr/local/lib/dapp-bin contract.sol
+solc --)" +
+		 g_argBinary + R"( -o /tmp/solcoutput dapp-bin=/usr/local/lib/dapp-bin contract.sol
 
-General Information)").c_str(),
+General Information)")
+			.c_str(),
 		po::options_description::m_default_line_length,
 		po::options_description::m_default_line_length - 23
 	);
-	desc.add_options()
-		(g_argHelp.c_str(), "Show help message and exit.")
-		(g_argVersion.c_str(), "Show version and exit.")
-		(g_strLicense.c_str(), "Show licensing information and exit.")
-	;
+	desc.add_options()(g_argHelp.c_str(), "Show help message and exit.")(
+		g_argVersion.c_str(),
+		"Show version and exit."
+	)(g_strLicense.c_str(), "Show licensing information and exit.");
 
 	po::options_description inputOptions("Input Options");
-	inputOptions.add_options()
-		(
-			g_argBasePath.c_str(),
-			po::value<string>()->value_name("path"),
-			"Use the given path as the root of the source tree instead of the root of the filesystem."
-		)
-		(
-			g_argAllowPaths.c_str(),
-			po::value<string>()->value_name("path(s)"),
-			"Allow a given path for imports. A list of paths can be supplied by separating them with a comma."
-		)
-		(
-			g_argIgnoreMissingFiles.c_str(),
-			"Ignore missing files."
-		)
-		(
-			g_argErrorRecovery.c_str(),
-			"Enables additional parser error recovery."
-		)
-	;
+	inputOptions.add_options()(
+		g_argBasePath.c_str(),
+		po::value<string>()->value_name("path"),
+		"Use the given path as the root of the source tree instead of the root of the filesystem."
+	)(g_argAllowPaths.c_str(),
+	  po::value<string>()->value_name("path(s)"),
+	  "Allow a given path for imports. A list of paths can be supplied by separating them with a "
+	  "comma.")(g_argIgnoreMissingFiles.c_str(), "Ignore missing files.")(
+		g_argErrorRecovery.c_str(),
+		"Enables additional parser error recovery."
+	);
```

Comment:
It completely butchers `boost::program_options` option definitions. Granted, `program_options` has this weird syntax where you chain `operator()` calls, but still, the result is incomprehensible to me while the original shows options in a pretty clear way.